action-server/db/reminder.py

```diff
@@ -1,11 +1,59 @@
+import logging
+from typing import Any, Dict, Text
+
+import pytz
 from sqlalchemy import Boolean, CheckConstraint, Column, DateTime, Integer, String, func
 from sqlalchemy.dialects.postgresql import JSONB
 
 from .base import Base
 
-# TODO: We might want to use auto-mapping instead of re-defining the actual tables.
+logger = logging.getLogger(__name__)
+
+
+FIRST_NAME_ATTRIBUTE = "first_name"
+PHONE_NUMBER_ATTRIBUTE = "phone_number"
+LANGUAGE_ATTRIBUTE = "language"
+PROVINCE_ATTRIBUTE = "province"
+AGE_OVER_65_ATTRIBUTE = "age_over_65"
+PRECONDITIONS_ATTRIBUTE = "preconditions"
+HAS_DIALOGUE_ATTRIBUTE = "has_dialogue"
+
+FIRST_NAME_SLOT = "first_name"
+PHONE_NUMBER_SLOT = "phone_number"
+LANGUAGE_SLOT = "language"
+PROVINCE_SLOT = "province"
+AGE_OVER_65_SLOT = "age_over_65"
+PRECONDITIONS_SLOT = "preconditions"
+HAS_DIALOGUE_SLOT = "has_dialogue"
+METADATA_SLOT = "metadata"
+TIMEZONE_METADATA_PROPERTY = "timezone"
```

Comment:
@LaurenceDupont new slots constants that will need to be extracted in your branch